python/ray/serve/_private/logging_utils.py

```diff
@@ -82,21 +92,12 @@ def format(self, record: logging.LogRecord) -> str:
         record_attributes = copy.deepcopy(record.__dict__)
         record_format[SERVE_LOG_LEVEL_NAME] = record.levelname
         record_format[SERVE_LOG_TIME] = self.asctime_formatter.format(record)
-
-        if SERVE_LOG_REQUEST_ID in record_attributes:
-            record_format[SERVE_LOG_REQUEST_ID] = record_attributes[
-                SERVE_LOG_REQUEST_ID
-            ]
-        if SERVE_LOG_ROUTE in record_attributes:
-            record_format[SERVE_LOG_ROUTE] = record_attributes[SERVE_LOG_ROUTE]
-
-        if SERVE_LOG_APPLICATION in record_attributes:
-            record_format[SERVE_LOG_APPLICATION] = record_attributes[
-                SERVE_LOG_APPLICATION
-            ]
-
         record_format[SERVE_LOG_MESSAGE] = self.message_formatter.format(record)
 
+        for field in ServeJSONFormatter.ADD_IF_EXIST_FIELDS:
+            if field in record_attributes:
+                record_format[field] = record_attributes[field]
```

Comment:
That would require us to pass more things to the constructor, but I guess it makes more sense there since they won't change between each log messages. Let me update it :)

Reply:
These ones are just calling `get_runtime_context()` anyways

Reply:
Oh that's right. Let me just call those in init 😅